.github/workflows/code-check-cloud-storage.yml

```diff
@@ -93,14 +93,15 @@ jobs:
         run: |
           cargo install just --force
           cargo install sqlx-cli --force
+          cargo install cargo-nextest --force
```

Comment:
@seanaye can you also fix the caching as part of this PR. The fact we are getting cache misses means the more dependencies we install in our testing toolchain the worse things get